location/templates/location.html

```diff
@@ -37,209 +35,258 @@
             .openPopup();
     });
 </script>
+
```

Comment:
Can we move this style block out of the `{% block scripts %} ... {% endblock %}` and into the `{% block styles %}` block?  In a previous PR I split the scripts and styles blocks into separate ones so that the styles could be loaded at the top of the page and the scripts loaded at the end for better performance.  With this one there's a slight lag when the page is loading where it looks really weird.  

The `{% block styles %}` block is at line 7-9 in `location.html`.

Reply:
Good point! Fixed it.